manifest.md

```diff
@@ -133,16 +141,88 @@ Unlike the [image index](image-index.md), which contains information about a set
+When the `ScratchDigestSHA256` is used, the media type SHOULD end with `+oci` to differentiate the descriptor from one pointing to content.
 
 ```json,title=SCRATCH%20config&mediatype=application/vnd.oci.descriptor.v1%2Bjson
 {
-  "mediaType": "application/vnd.oci.example+json",
+  "mediaType": "application/vnd.example+oci",
   "size": 2,
   "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a"
 }
 ```
 
+## Guidelines for Artifact Usage
+
+Content other than OCI container images MAY be packaged using the image manifest.
+When this is done, the `config.mediaType` value MUST be set to a value specific to the artifact type.
+For portability, if the artifact does not have a content for the config blob the [SCRATCH](#example-of-a-scratch-config-or-layer-descriptor) descriptor SHOULD be used with a `+oci` suffix on the `mediaType`.
+If the artifact does not need layers, a single layer SHOULD be included with a non-zero size.
+The suggested content for an unused layer is the [SCRATCH](#example-of-a-scratch-config-or-layer-descriptor) descriptor.
+
+Here is an example manifest for a typical artifact:
+
+```json,title=Artifact%20with%20config&mediatype=application/vnd.oci.image.manifest.v1%2Bjson
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "config": {
+    "mediaType": "application/vnd.example.config.v1+json",
+    "digest": "sha256:5891b5b522d5df086d0ff0b110fbd9d21bb4fc7163af34d08286a2e846f6be03",
+    "size": 123
+  },
+  "layers": [
+    {
+      "mediaType": "application/vnd.example.data.v1.tar+gzip",
+      "digest": "sha256:e258d248fda94c63753607f7c4494ee0fcbe92f1a76bfdac795c9d84101eb317",
+      "size": 1234
+    }
+  ]
+}
+```
+
+Here is an example manifest for a simple artifact without content in the config, using the scratch descriptor:
+
+```json,title=Artifact%20without%20config&mediatype=application/vnd.oci.image.manifest.v1%2Bjson
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "config": {
+    "mediaType": "application/vnd.example+type+oci",
+    "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
+    "size": 2
+  },
+  "layers": [
+    {
+      "mediaType": "application/vnd.example+type",
+      "digest": "sha256:e258d248fda94c63753607f7c4494ee0fcbe92f1a76bfdac795c9d84101eb317",
+      "size": 1234
+    }
+  ]
+}
+```
+
+Here is an example minimal manifest for an artifact, with the content of both blobs set to the scratch descriptor:
+
+```json,title=Minimal%20artifact&mediatype=application/vnd.oci.image.manifest.v1%2Bjson
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "config": {
+    "mediaType": "application/vnd.example+type+oci",
+    "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
+    "size": 2
+  },
+  "layers": [
+    {
+      "mediaType": "application/vnd.example+type+oci",
```

Comment:
Setting the mediaType of the ScratchData layer to the same as the config blob here is acceptable. And people can argue about whether or not the content really is that media type, it kind of depends on what that mediaType's spec says.

But as this layer mediaType is not expected to be used to drive any sort of behaviour, unlike the config mediaType, why not use `applcation/octet-stream`, because when is something not an octet stream?